### PR TITLE
Piggybacking for Humans [DNM]

### PR DESCRIPTION
**Pull request by RMagnum**
This (maybe) ports piggybacking to RP. Working with another person now to hopefully finish this, eventually.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_vr.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 /datum/riding/dogborg
 	keytype = /obj/item/weapon/material/twohanded/fluff/riding_crop // Crack!
 	nonhuman_key_exemption = FALSE	// If true, nonhumans who can't hold keys don't need them, like borgs and simplemobs.
-	key_name = "a riding crop"		// What the 'keys' for the thing being rided on would be called.
+	key_name = "a riding crop"		// What the 'keys' for the thing being rode on would be called.
```
Comment (nik707) [inline code/modules/mob/living/silicon/robot/robot_vr.dm]: ridden

**Comment by nik707** [comment]
You can test this by getting a second person connected to a test server, but also this would need modularization

File: code/modules/mob/living/silicon/robot/robot_vr.dm (hunk 1 above)
Comment (RMagnum) [reply code/modules/mob/living/silicon/robot/robot_vr.dm]: oh god oh fuck

Comment (C0rva1r) [reply code/modules/mob/living/silicon/robot/robot_vr.dm]: OOF

**Pull request closed without merging**